tests/platform_tests/thermal_control_test_helper.py

```diff
@@ -303,7 +303,8 @@ def __enter__(self):
         thermal control daemon to make it effect.
         :return:
         """
-        self.dut.command('mv -f {} {}'.format(self.thermal_policy_file_path, self.thermal_policy_file_backup_path))
+        if os.path.exists(self.thermal_policy_file_path):
+            self.dut.command('mv -f {} {}'.format(self.thermal_policy_file_path, self.thermal_policy_file_backup_path))
```

Comment:
Can we add a logger.warning to indicate we encountered this case here where the thermal_policy file is not there.
The warning logging will not cause the testcase to fail bit to provide a record of this occured where the file is missing in case the file should be there but not...

Reply:
added warning message